core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -471,14 +471,28 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
         val updates = updateEntityAndWorkspace(workflowRecord.workflowEntityId.map(id => Some(entitiesById(id))).getOrElse(None), workspace, boundExpressions)
 
         val (optEntityUpdates, optWs) = updates
-        optEntityUpdates foreach { update: WorkflowEntityUpdate =>
-          logger.debug(s"Updating ${attributeCount(update.upserts)} attribute values for entity ${update.entityRef.entityName} of type ${update.entityRef.entityType} in ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}. First 100: ${update.upserts.take(100)}")
-        }
-        optWs foreach { workspace: Workspace =>
-          logger.debug(s"Updating ${attributeCount(workspace.attributes)} attribute values for workspace in ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}. First 100: ${workspace.attributes.take(100)}")
-        }
 
-        Left(updates)
+        val entityAttributeCount = optEntityUpdates map { update: WorkflowEntityUpdate =>
+          val cnt = attributeCount(update.upserts)
+          logger.debug(s"Updating $cnt attribute values for entity ${update.entityRef.entityName} of type ${update.entityRef.entityType} in ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}. ${safePrint(workspace.attributes)}")
```

Comment:
Yes, I was the author of the previous line also and because virtually all lists contain repeitive data 10 is more than enough to figure out what's happening